services/organization_service/package.json

```
@@ -6,6 +6,8 @@
   "scripts": {
     "start": "node server.js",
```

Biome (2.1.2)

[error] The key start was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)